cosmwasm/utils.js

```diff
@@ -121,15 +122,38 @@ const getAmplifierContractConfig = (config, { contractName, chainName }) => {
     return { contractBaseConfig, contractConfig };
 };
 
-const usesGovernanceBypass = (config, contractName, chainName) => {
+const validateGovernanceMode = (config, contractName, useGovernance, chainName) => {
+    if (!useGovernance) {
+        return;
+    }
+
```

Comment:
```suggestion
const validateGovernanceMode = (config, contractName, chainName) => {
```
this arg seems unnecessary